docs/modules/game-object-creation/standard-usage.md

```diff
@@ -0,0 +1,42 @@
+<%(Toc)%>
+# Setup basic spawning
+
+## Setup your GameObject prefabs
+
+The [`GameObjectCreatorFromMetadata`]({{urlRoot}}/api/game-object-creation/game-object-creator-from-metadata) class is the default implementation for spawning GameObjects for your SpatialOS entities. This class expects your GameObject prefabs to be in a particular place in your project.
+
+The `GameObjectCreatorFromMetadata` uses the `entity_type` field in the `Metadata` component along with the worker instance's type to determine which prefab to spawn.
+
+<%(Callout message="You can set the `entity_type` field in the `Metadata` component when declaring your [EntityTemplate]({{urlRoot}}/reference/concepts/entity-templates.md)")%>
+
+For a given worker type `<worker-type>` it will look for a prefab at:
+
+```text
+Resources/Prefabs/<worker-type>/<Metadata.entity-type>
```

Comment:
should we mention the Common folder here as well?
as it's not clear how you derived the rules stated afterwards

Reply:
Yes good call